website/docs/getting-started/features.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Features
+sidebar_position: 2
+---
+
+# Features provided by odo
+
+odo follows "create and push" workflow for almost everything. It means, as a user, when you "create" something the information (or manifest) is stored in a configuration file, and then upon doing a "push" it gets created  on the Kubernetes cluster.
```

Comment:
You could use the vocabulary used in Kubernetes: declarative, reconciliation

That way, people already using Kubernetes will understand that odo is a natural extension of the Kubernetes tools.

Reply:
OK. I can't figure out how to put that in context of odo.

odo uses devfile.yaml which is declarative just like k8s yaml manifests. But we don't want users to know much/any of devfile, so I'm not sure how to use "declarative" in context of odo.

As for "reconciliation" I'm totally clueless about how to use it in the context of odo.